packages/experiments/src/components/Tile/Tile.scss

```diff
@@ -402,9 +432,43 @@
       transition: opacity 0.2s linear;
       opacity: 0;
     }
+
+    .name {
+      opacity: 0;
+      transition: transform 0.25s ease-out, opacity 0.2s linear, color 0.2s linear;
+      transform: translateY(1rem);
+    }
+
+    .activity {
+      opacity: 0;
+      transition: transform 0.25s ease-out, opacity 0.2s linear, color 0.2s linear;
+      transform: translateY(1rem);
+    }
+  }
+
+  &:hover,
+  &.selected {
+    .name {
+      opacity: 1;
+      transform: translateY(0px);
+    }
+
+    .activity {
+      opacity: 1;
+      transform: translateY(0px);
+    }
   }
 
   &.showBackground {
```

Comment:
The default mode should continue always to show the metadata as it did before. However, you can add a new flag to make it hide the nameplate information until hover or selection.

Reply:
Fair enough. I've added a `nameplateOnlyOnHover` property to enable the transition effect. Leaving support for multiple lines should not affect existing users.